schema/core.xsd

```diff
@@ -419,6 +419,7 @@
     </xs:attribute>
     <xs:attribute name="shows-scroll-indicator" type="xs:boolean" />
     <xs:attribute name="content-container-style" type="hv:styleList" />
+    <xs:attribute name="dismiss-keyboard-on-drag" type="xs:boolean" />
```

Comment:
Looks like rn supports another mode `interactive` for ios. What do you think about keeping the the prop same same(`keyboard-dismiss-mode`) and making it an enum instead?

Reply:
I originally had an enum instead of boolean, but couldn't get the interactive mode to work in Expo, which led me to refactor with a boolean instead

Reply:
ok, imo I think we can still have it as an enum with only 2 options for now. That will give us flexibility in the future, but I will leave it up to you.

Reply:
If there's any chance of additional options, I'd definitely go with enum, even if only two values

Reply:
I'll defer to the general opinion here and will revert back to using an enum

Reply:
@hgray-instawork @palaparthi I've updated this to use an enum in [b9c89ef](https://github.com/Instawork/hyperview/pull/678/commits/b9c89ef4f6b04e94232a711eba43c50a91b240f4)

Reply:
@flochtililoch is interactive working on expo now? If not maybe we shouldn't add it?

Reply:
No, but I also have not added any example in the Hyperview repo to use it since it didn't work with Expo. I don't know yet if it work with bare RN app.